File: LND/C/Assignments/Assignment2/str_rot.c
#include "header.h"

int str_rot (char *buf, char *rbuf)
{
	char ch;
	int len, i, j, count=0;
	for(len = 0; buf[len] != '\n'; len++);
	ch=buf[len-1];
	for(i = (len - 1); i > 0; i--)
	{
		buf[i] = buf[i-1];
	}
	buf[0]=ch;
	buf[len] = '\0';
//	printf("%s\n",buf);

	for(i = 0, j = 0; buf[i]; i++, j++) {
		if(buf[i]==rbuf[j]) {
			count=count+1;
		}
	}
//	printf("%d\n",count);
//	printf("%d\n",len);
	if(count == len)
		return 1;
	else
		return 0;

}

File: PYTHON/anfal's_python/ass/python/assignments/strrot.py
#!/usr/bin/python

"""
	Script : strrot.py
	Author : Bhavya
	Date : Aug-31-2017
	Purpose :String rotation
"""

string1=raw_input("enter string:")
count=int(raw_input("enter number of times rotation:"))

if len(string1)!=count:
	for i in range(0,count):
		string1=string1[-1]+string1[0:len(string1)-1]

print string1
